src/notation/internal/notationinteraction.cpp

```diff
@@ -3844,36 +3844,43 @@ void NotationInteraction::addGraceNotesToSelectedNotes(GraceNoteType type)
 
 bool NotationInteraction::canAddTupletToSelectedChordRests() const
 {
-    for (ChordRest* chordRest : score()->getSelectedChordRests()) {
+    mu::engraving::TDuration dur512(mu::engraving::DurationType::V_512TH);
+    mu::engraving::TDuration durMeasure(mu::engraving::DurationType::V_MEASURE);
+
+    for (const ChordRest* chordRest : score()->getSelectedChordRests()) {
         if (chordRest->isGrace()) {
             continue;
         }
 
-        if (chordRest->durationType() < mu::engraving::TDuration(mu::engraving::DurationType::V_512TH)
-            && chordRest->durationType() != mu::engraving::TDuration(mu::engraving::DurationType::V_MEASURE)) {
-            return false;
+        mu::engraving::TDuration chordRestDur = chordRest->durationType();
+        if (chordRestDur < dur512 && chordRestDur != durMeasure) {
+            continue;
         }
+
+        return true;
```

Comment:
I don't fully understand this change. It looks like the new logic says: "if any of the notes is _not_ too short, allow a tuplet". 
The old logic seems _more_ conservative to me: "if any of the notes _is_ too short, disallow a tuplet".